test/RoadRegistry.Tests/Product/Projections/Customizations.cs

```diff
@@ -0,0 +1,267 @@
+// namespace RoadRegistry.Product.Projections
```

Comment:
what's up with this file?

Reply:
I'll remove it. Duplication was not necessary.